common/protos/xyz/block/ftl/schema/v1/schema.proto

```diff
@@ -240,6 +240,12 @@ message Event {
   }
 }
 
+message Export {
+  optional Position pos = 1;
+  bool exported = 2;
+  bool realm_exported = 3;
+}
```

Comment:
I think call this "visibility" as we will likely extend this later. I'd also combine these fields into an enum, as it doesn't make any sense for "exported" to be false" and "realm_exported" to be true. eg.

```suggestion
enum Scope {
  SCOPE_NONE = 0;
  SCOPE_MODULE = 1;
  SCOPE_REALM = 2;
}
  
message Visibility {
  optional Position pos = 1;
  Scope scope = 2;
}
```

Reply:
"Scope" might be too generic for a top level name

Reply:
Renamed this to "Visibility"